nightingale_dispatcher/src/nightingale_dispatcher/mission_planner.py

```diff
@@ -38,6 +41,11 @@ def __init__(self):
 
         self.phases = queue.Queue()
 
+        self.fallback_bin_goal = Point()
```

Comment:
hmm this adds yet another hard-coded point. we are starting to have quite a lot of thse in several places, and theyre hard to keep track of and update. do u think itd be reasonable to delete this fallback_bin_goal Point and have the robot call extend_restock() if the dynamic detection fails? extend restock actually has this point defined inside of iirc

Reply:
sure sounds good